javascript/javascript3/week3/canvas/main.js

```diff
@@ -0,0 +1,63 @@
+const canvas = document.querySelector("canvas");
+
+canvas.width = window.innerWidth;
+canvas.height = window.innerHeight;
+
+const context = canvas.getContext("2d");
+
+class Circle {
+  constructor(x, y, r, startAngle, endAngle, fillColor) {
+    this.x = x;
+    this.y = y;
+    this.r = r;
+    this.startAngle = startAngle;
+    this.endAngle = endAngle;
+    this.fillColor = fillColor;
+  }
+  draw() {
+    context.beginPath();
+    context.fillStyle = "#" + Math.floor(Math.random() * 16777215).toString(16);
+
+    context.arc(
+      this.x,
+      this.y,
+      this.r,
+      this.startAngle,
+      this.endAngle,
+      this.fillColor
```

Comment:
The fillColor parameters is not supported by the arc function: https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/arc

Instead, you could use the fillColor parameter in the ``` context.fillStyle```